command-line-local-shortener/main.go

```diff
@@ -128,9 +95,24 @@ func routerSetup() *gin.Engine {
 }
 
 func main() {
+
+	jsonFile, err := os.Open("urlmap.json")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Conver this to logrus just like in [brad-test](https://github.com/codeday-labs/golang-url-shortener/blob/brad-test/brad-test/main.go#L29)

Reply:
oh right